src/data/datatools.ts

```diff
@@ -74,6 +77,18 @@ export function parseAndInsertStringWithFieldContent(
   defaultValue?: string,
   fieldValueToStringFn?: (fieldName: string, valueObject: IFieldValue) => string,
   instance?: IInstanceDetails,
+  riskMetrics?: { [riskName: string]: number },
+): string | undefined;
+export function parseAndInsertStringWithFieldContent(
```

Comment:
the declaration equals the last overload signature, please remove the unnecessary overload signature.